src/SDKs/ApplicationInsights/DataPlane/Data.ApplicationInsights.Tests/Data.ApplicationInsights.Tests.csproj

```diff
@@ -14,7 +14,9 @@
   </ItemGroup>
   <ItemGroup>
```

Comment:
Please replace `VersionPrefix` with `Version`
Also add `<TreatWarningsAsErrors>true</TreatWarningsAsErrors>` to the same `PropertyGroup`

Reply:
Fixed